app/src/main/java/net/squanchy/navigation/view/BottomNavigationPagerAdapter.java

```diff
@@ -6,26 +6,25 @@
 
 import com.novoda.viewpageradapter.ViewPagerAdapter;
 
-import net.squanchy.schedule.ScheduleView;
-
 public class BottomNavigationPagerAdapter extends ViewPagerAdapter<View> {
 
-    private static final int SCHEDULE_POSITION = 0;
-    private static final int FAVOURITES_POSITION = 1;
-    private static final int TWEETS_POSITION = 2;
-    private static final int VENUE_POSITION = 3;
+    public static final int SCHEDULE_POSITION = 0;
```

Comment:
We would still need the positions to be public because they are used in the BottomNavigationActivity